cli/pom.xml

```diff
@@ -66,9 +66,18 @@
             <version>0.9.2</version>
         </dependency>
         <dependency>
-            <groupId>jline</groupId>
+            <groupId>org.jline</groupId>
             <artifactId>jline</artifactId>
         </dependency>
+        <dependency>
+            <groupId>net.java.dev.jna</groupId>
+            <artifactId>jna</artifactId>
+        </dependency>
```

Comment:
Hi, what is this dependency use for?

Reply:
To access the JVM's main terminal under the Windows system for Jline3 to provide advanced features.

Please see:
![image](https://user-images.githubusercontent.com/20107543/143218637-57dfc92b-5301-4967-93da-11a7e1ea8c65.png)